stdlib/3/builtins.pyi

```diff
@@ -196,15 +196,15 @@ class float:
     def __truediv__(self, x: float) -> float: ...
     def __mod__(self, x: float) -> float: ...
     def __divmod__(self, x: float) -> Tuple[float, float]: ...
-    def __pow__(self, x: float) -> float: ...
+    def __pow__(self, x: float) -> complex: ...
```

Comment:
This should have an overload for when `x` is an `int`, since it will never be complex in that case. See #2444 for an example. Apart from that this PR looks good to me.

Reply:
Sure. On it.

Reply:
Looks good now, @srittau?

Reply:
Yeah, sure.
Didn't quite get it.